src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello, I'm Duke!\n" + "What would you like to do today?");
+        Scanner response = new Scanner(System.in);
+        String line = response.nextLine();
+        int counter = 0;
+        String[] items = new String[101]; // 1-based
+        String[] check = new String[101]; // 1-based
+
+        while (!(line.equals("bye"))) {
```

Comment:
Perhaps you could use a switch case so that it is easier when there are more commands?